README.md

```diff
@@ -36,4 +36,4 @@ Każdy użytkownik może po otwarciu aplikacji sprawdzić jaki jest jego sperson
 
 Aplikacja docelowo powinna być zainstalowana przez każdego obywatela. Rozpoczynamy budowę kultury używania aplikacji np. poprzez pokazywanie sobie nawzajem swojego zielonego statusu podczas powitania.
 
-Ze względu na zrozumiały opór społeczny przed permanentną inwigilacją obywateli, kładziemy duży nacisk na zadbanie o ochronę prywatności. Kod aplikacji jest upubliczniony (open source) i może być zaudytowany przez ekspertów.
+Ze względu na zrozumiały opór społeczny przed permanentną inwigilacją obywateli, kładziemy duży nacisk na zadbanie o ochronę prywatności. Kod aplikacji jest upubliczniony (open source) i może być poddany audytowi przez osoby trzecie.
```

Comment:
Powinno być napisane nie jak jest proponowane "osoby trzecie", ale pozostawione "ekspertów" i dodane słowa "bezpieczeństwa". Najlepiej gdyby audytu dokonał Niebezpiecznik wraz z Panoptykonem. 

Osoba trzecia to jest Jan Kowalski z Błotnej 50.

Reply:
To może "przez każdego zainteresowanego"? Pisząc "ekspertów" co masz na myśli? Kto nadaje ten tytuł eksperta? Dlaczego laik nie może dokonać samodzielnie audytu bez wcześniejszego awansu na eksperta?

Reply:
audytorów bezpieczeństwa? Poza tym trzeba pisać przez kogo? Może użyć dobrze znanego sformułowania "audyt bezpieczeństwa"?

Reply:
"poddany audytowi przez audytorów bezpieczeństwa" - brzmi jak "masło maślane". I znowu rodzi pytanie - jakich audytorów? Rządowych? Z ramienia ONZ?